fiat-core/src/main/java/com/netflix/spinnaker/fiat/model/UserPermission.java

```diff
@@ -46,7 +47,11 @@ public View getView() {
   @Data
   public class View {
     String name = UserPermission.this.id;
-    Map<String, Object> resources = ImmutableMap.of(
+    Set<ServiceAccount.View> serviceAccounts = UserPermission.this.serviceAccounts
+        .stream()
+        .map(ServiceAccount::getView)
+        .collect(Collectors.toSet());
+    Map<String, Object> cloudResources = ImmutableMap.of(
```

Comment:
Maybe type this stronger than `Object`? I think we can have a general `ProtectedResource` superclass that things like `Application`, etc extend so we at least a bit of static typing.